### PR TITLE
Feature/phenotype

### PR DESCRIPTION
**Pull request by adamjohnwright**
I have created and tested the Phenotype URI. This URL to my knowledge creates identical JSON as Catalyst does.

In order to make sure everything is correct I have gone through a substantial amount of the code with @mgrbyte  and will be going over this code with @sibyl229.

Thanks everybody for the help!

@tharris This is fairly close we just have to go through the proper testing procedure with the pull request and with getting a curator or two to check over the results.

### CODE REVIEW

File: src/datomic_rest_api/rest/gene.clj [hunk 1]
```diff
@@ -533,12 +533,13 @@
...
                             (keys rnai-phenos)))]
     (->>
-      (for [pid phenos :let [pheno (entity db pid)]]
-        (let [pcs (get-pato-combinations db pid rnai-phenos var-phenos not?)]
-          (if (nil? pcs)
-            (phenotype-table-entity db pheno nil nil pid var-phenos rnai-phenos not?)
-            (for [[pato-key entity] pcs]
-              (phenotype-table-entity db pheno pato-key entity pid var-phenos rnai-phenos not?)))))
+      (flatten
+        (for [pid phenos :let [pheno (entity db pid)]]
```
Comment (mgrbyte) [inline src/datomic_rest_api/rest/gene.clj]: I think the style convention for `:let` in forms brings it onto its own line e.g:
```clojure
(for [x xs
      :let [n-xs (count xs)]]
    (do-something x))
```

**Review by mgrbyte — COMMENTED** [review]
I think this formatting of `where` clauses is fine,
however I prefer the following:
```clojure
:where
 [clause-1]
 ....
 [clause-n]
```
since this helps avoids long lines.

(General comment on c4da5f0)

File: src/datomic_rest_api/get_handler.clj [hunk 2]
```diff
@@ -28,15 +28,23 @@
                     <h5>Widgets</h5>
                     <ul>
                        <li><a href=\"./rest/widget/\">/rest/widget/</a></li>
+                       <li><a href=\"./rest/field/\">/rest/field/</a></li>
                     <ul>
                   </html>")
+     (GET "/rest/fiedl/" [] "<html>
```
Comment (mgrbyte) [inline src/datomic_rest_api/get_handler.clj]: I assume there's a typo here, should be: 
(GET "/rest/field" ...

File: src/datomic_rest_api/rest/gene.clj (hunk 1 above)
Comment (adamjohnwright) [reply src/datomic_rest_api/rest/gene.clj]: I have pushed the fix for this

File: src/datomic_rest_api/get_handler.clj (hunk 2 above)
Comment (adamjohnwright) [reply src/datomic_rest_api/get_handler.clj]: This was a type - fixed

**Review by mgrbyte — APPROVED** [review]
:+1:

**Pull request merged**